.github/workflows/rebuild-latest-stable-image.yml

```diff
@@ -0,0 +1,57 @@
+name: Rebuild latest release image version
+
+on:
+  schedule:
+    - cron: 0 0 * * 1
+  workflow_dispatch:
+  push:
+    branches:
+      - "main"
+
+jobs:
+  rebuild:
+    name: Rebuild latest release image version
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout main fcrepo repository"
+        uses: actions/checkout@v3
+        with:
+          repository: "fcrepo/fcrepo"
+
+      - name: "Checkout fcrepo Docker repository"
+        uses: actions/checkout@v3
+        with:
+          path: "fcrepo-docker"
```

Comment:
Do we need to specify the repository here? Isn't that checked out by Github pipelines automatically?

Reply:
no, you always need a manual `checkout` on GitHub Actions. But it defaults to the current repository.